lib/src/main/kotlin/com/lemonappdev/konsist/core/provider/KoTextProviderCore.kt

```diff
@@ -11,6 +12,8 @@ internal interface KoTextProviderCore : KoTextProvider, KoBaseProviderCore {
 
     override fun print(prefix: String?) {
         prefix?.let { println(it) }
-        kotlin.io.print(toString())
+
+        val text = if (this is KoNameProvider) name else toString()
```

Comment:
Does this condition have to be duplicated in [KoBaseProviderListExt.kt](https://github.com/LemonAppDev/konsist/pull/443/files#diff-0b283bcc5757f88d85e8738116d1902f4c84c449ef061416a3b2bbf0d66f824a)?